code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -464,6 +464,13 @@ Pass the desired type path itself, declaring a temporary var beforehand is not r
 			continue //The current element failed assessment, move on to the next.
 		return final_result
 
+/mob/living/simple_animal/bot/proc/check_bot(targ)
```

Comment:
`targ` is a bit non-descriptive. Just using `target` is fine. Also if you know the type it has then it's best to include it
```suggestion
/mob/living/simple_animal/bot/proc/check_bot(atom/target)
```
Be sure to change the code below as well to include the name change